lmdeploy/cli/serve.py

```diff
@@ -164,11 +164,14 @@ def add_parser_api_server():
         max_prefill_token_num_act = ArgumentHelper.max_prefill_token_num(pt_group)
         quant_policy = ArgumentHelper.quant_policy(pt_group)
         ArgumentHelper.dp(pt_group)
-        ArgumentHelper.dp_rank(pt_group)
         ArgumentHelper.ep(pt_group)
         ArgumentHelper.enable_microbatch(pt_group)
         ArgumentHelper.role(pt_group)
         ArgumentHelper.migration_backend(pt_group)
+        # multi-node serving args
+        ArgumentHelper.node_rank(parser)
+        ArgumentHelper.num_nodes(parser)
+        ArgumentHelper.base_gpu_id(parser)
```

Comment:
Can we use `ngpus_per_node` and re-implement the function based on `ngpus_per_node`?
`base_gpu_id` is not not immediately clear

Reply:
base_gpu_id is for settig the first GPU to use, same as in sglang https://docs.sglang.ai/backend/server_arguments.html#other-runtime-options